classes/system/galera/server/database/aodh.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
```

Comment:
@ityaptin, see this commit where it only adds to classes.system.galera.server.database... I would remove the system.mysql.client.database.aodh class for clarity.